components/eamxx/src/doubly-periodic/CMakeLists.txt

```diff
@@ -49,11 +49,10 @@ set_target_properties(dp PROPERTIES
   Fortran_MODULE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/modules
 )
 target_include_directories(dp PUBLIC
-  ${CMAKE_CURRENT_SOURCE_DIR}/../share
+  ${CMAKE_CURRENT_SOURCE_DIR}/../physics/share
```

Comment:
This line should not be needed, right? linking the `physics_share` lib should bring in all its include dirs...